src/components/Filter.js

```diff
@@ -1,14 +1,28 @@
 import React from "react";
-import { generateFilter } from "../functions/generateFilter";
+import { Dropdown } from "semantic-ui-react";
+import { generateFilterOptions } from "../functions/generateFilterOptions";
 
-export function Filter({ styling, filterColumns, onChange, dataSet }) {
+export const Filter = ({ styling, filterColumns, onChange, dataSet }) => (
```

Comment:
name should be plural. we render more than one filter

Reply:
b37ea0f97a908c73eeae550dfd0e12c1e9091628